docs/test-example.md

```
@@ -151,3 +151,17 @@ print("phi:", fvs[1][:5])
 phi:   [4.30053475 2.58923118 4.61933201 6.0707641  0.62762944]
 ```
+
+## Compute Single Fréchet Vector
+```python
+%%time
+fvs_single = polymv.frechet_single(fvs[0], fvs[1])
+print("(Theta Phi) ", fvs_single)
+```
+
+**Output:**
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)